packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -8,9 +8,28 @@ import PropTypes from 'prop-types';
  * This component expects all necessary scripts for BookReader to be pre-loaded onto the page before use.
  * Please see https://github.com/internetarchive/bookreader for more instructions & examples.
  *
+ * If jsia is specified then it adds Internet Archive specific stuff via the BookreaderJSIA function
+ * the script BookreaderJSIA should previously have been loaded.
+ *
+ * TODO - someone could merge all or part of BookreaderJSIA into here and obsolete BookreaderJSIA ideally without changing the interface
+ * TODO - except Isa doesnt want the Archive's lending logic in here.
+ * Note this component is used by dweb-archive for offline and dweb versions.
+ *
  * global: BookReader
  * Creates a global: window.br (which Search will need)
  *
+ * <BookReaderWrapper
+ *   jsia={ options returned by JSIA call}  optional
+ *   options={ options to override default bookreader options}
+ *   />
+ *
+ * Note will almost certainly need a AJS.theatresize() in caller after this.
+ *
+ * Questions for ISA
+ * - why are defaultStartLeaf and titleLeaf not in defaultOptions ? I also moved them to start of list so can be overridden
+ * - I reinstated IABookReaderMessageWrapper (just for the JSIA case) which I'm guessing is used by lending tools to put their message
+ * - Any reason options.onePage.autofit is defaulting to 'height' not 'auto' (which is what I've seen before
+ * - What is going on with the getPgeURI definition, a comment would be good
```

Comment:
@mitra42 great questions, thanks for making this a PR to chat about.

One of the issues we have right now, is that the default params are actually custom for the liner notes in the audio items. Are you able to successfully trump them?

Reply:
Sorry, I don't understand that question, what I meant is that you've got a mechanism to setup defaultOptions and then to override them with "options" set in the properties of the component. 
a) Why pull defaultStartLeaf and titleLeaf out of that mechanism rather than having the lines
```
defaultStartLeaf: 0
titleLeaf: 0
``` 
be inside the definition of defaultOptions.   I didn't change this,. 

And b: if you must have them seperate then why put them first in this structure:
```
      ...defaultOptions,
      ...options,
      defaultStartLeaf: 0,
      titleLeaf: 0,
```
which means that they can't be overridden by options, rather than last i.e .
```
    const fullOptions = {
      defaultStartLeaf: 0,
      titleLeaf: 0,
      ...defaultOptions,
      ...options,
    };
```
So that if specified in the component's properties e.g. <BookReaderWrapper titleLeaf:10> it will override your default.

Reply:
@mitra42 the current options listed is specifically for the liner notes display.

> So that if specified in the component's properties e.g. it will override your default.

that's the point.  It's specific to the liner notes use case.

I have created a ticket to parse it out as importable liner notes options for draw: https://github.com/internetarchive/iaux/issues/300

Reply:
OK - maybe I misunderstood that your request in https://github.com/internetarchive/iaux/issues/260 was for a merged wrapper to handle all the cases, in that case I think either
a) defaultStartLeaf and titleLeaf need to be moved to the consumer 
b) OR they can be here as defaults, but in that case they need to be able to be overridden - which they can't if they take precedence like this.   

I must be missing something, because I really can't understand how putting them at lowest precedence (bottom) changes your code at all unless your consumer is passing in some other value for titleLeaf  or defaultStartLeaf that you want this to override.

Reply:
> you want this to override.

yah, it was definitely an override. should we do #300 before hand or merge this in prior?

Reply:
Up to you - I don't understand #300 (because I don't know what is happening with liner notes) but sounds (to my ignorance) as something simple.